proto/dydxprotocol/clob/block_rate_limit_config.proto

```diff
@@ -13,8 +13,10 @@ message BlockRateLimitConfiguration {
   // configurations.
   //
   // Specifying 0 values disables this rate limit.
+  // Deprecated in favor of `max_short_term_orders_and_cancels_per_n_blocks`
+  // for v5.x onwards.
   repeated MaxPerNBlocksRateLimit max_short_term_orders_per_n_blocks = 1
-      [ (gogoproto.nullable) = false ];
+      [ (gogoproto.nullable) = false, deprecated = true ];
```

Comment:
should we just remove this? and mark 1 and 3 as reserved?

Reply:
we haven't removed any fields in production, but I feel like this is probably the way to do it